static/scss/answers/interactive-map/InteractiveMap.scss

```diff
@@ -238,7 +238,7 @@
       display: flex;
       align-items: center;
       border-top: var(--yxt-border-default);
-      color: var(--yxt-locationBias-text-color);
+      color: #737373;
```

Comment:
Could we make a variable for this?

Reply:
Sure, are you think adding another grey to the answers-variables, or adding a scss variable to InteractiveMap.sscs?

Reply:
I guess it depends on how often we feel the HHs will change this. If we think this is styling they will likely change, a CSS variable would probably be best. Otherwise, a scss variable in this file is fine.

Reply:
Since these are overriding SDK colors which weren't previously modifiable by the theme, i'll make it a scss variable in InteractiveMap.scss